modules/amxBidAdapter.js

```diff
@@ -302,8 +303,13 @@ export const spec = {
           bidRequestsCount: 0,
         };
 
+    const transactionId = deepAccess(
```

Comment:
What are you using this for? I can't make sense of this if `payload.a` is supposed to have any meaning at all when it gets to your server:

 - `auctionid` and `transactionId` are not meant to be mixed, picking them interchangeably means you cannot identify either the transaction nor the auction.
 - if you're ok with falling back to a randomly generated value, why not always use a randomly generated value? what's the reason for anything more complicated?
 - `ortb2.ext.tid` should be `ortb2.source.tid` instead (if the above is not a problem).